guides/common/modules/ref_template-sync-settings.adoc

```diff
@@ -12,6 +12,9 @@
 Case-insensitive.
 Snippets are not filtered.
 | *Force import* | No | If set to `Yes`, locked templates are overwritten during an import.
+| *HTTP proxy policy* | Global default HTTP proxy | Whether to use an HTTP proxy to synchronize templates.
+
+The `Custom HTTP proxy` value is available only if an HTTP proxy is configured under the selected organization and location and if your account has the *view_http_proxies* role assigned.
```

Comment:
Optional. I know we talked about this before but I don't really remember what the result of that conversation was so feel free to disregard if you feel like this is stating the obvious. 
```suggestion
The `Custom HTTP proxy` value is available only if an HTTP proxy is configured under the selected organization and location and if your account has the *view_http_proxies* role assigned.
If this option is selected, you will be prompted to select the HTTP proxy when syncing templates.
```

Reply:
I don't remember either :) but for the web UI, the information is part of the tooltip, and generally speaking, I think it is somewhat obvious that if you want to choose a value named `custom HTTP proxy`, a part of that process will be selecting said proxy :shrug: so no change.